extensions-core/datasketches/pom.xml

```diff
@@ -38,7 +38,12 @@
     <dependency>
       <groupId>com.yahoo.datasketches</groupId>
       <artifactId>sketches-core</artifactId>
-      <version>0.8.4</version>
+      <version>0.10.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.yahoo.datasketches</groupId>
+      <artifactId>memory</artifactId>
```

Comment:
why do we explicitly need to define memory dependency, would't it come based on sketches-core version:?